builtin/logical/ssh/backend_test.go

```diff
@@ -105,6 +105,13 @@ SeKWrUkryx46LVf6NMhkyYmRqCEjBwfOozzezi5WbiJy6nn54GQt
 `
 )
 
+type CertValidityValidationMethod int
+
+const (
+	ValidateTTL            CertValidityValidationMethod = 0
```

Comment:
This `ValidateTTL` enum is used to apply the existing validation logic for signed SSH certificates whose `ValidAfter` and `ValidBefore` were derived from `ttl`